app/src/main/java/io/homeassistant/companion/android/sensors/BluetoothSensorManager.kt

```diff
@@ -261,6 +342,33 @@ class BluetoothSensorManager : SensorManager {
         )
     }
 
+    fun updateBeaconMonitoringSensor(context: Context) {
```

Comment:
We should make sure none of the code below is reachable if the sensor is not enabled. May also want to do something similar with `updateBeaconMonitoringDevice()` up above. We do a similar check with other sensors so the code only runs if the sensor is enabled.